### PR TITLE
Updating d2l-navigation to new version.

### PR DESCRIPTION
**Pull request by CarolBusman**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -208,9 +208,9 @@
       "d2l-localize-behavior": "1.1.3",
       "d2l-menu": "1.1.0",
       "d2l-more-less": "4.3.1",
-      "d2l-my-courses": "5.7.22",
+      "d2l-my-courses": "5.7.23",
```
Comment (m6jones) [inline bower.json]: This is good! :) Serge update for d2l-my-courses.

Comment (m6jones) [reply bower.json]: Remove this version, I found the issue.

**Comment by svanherk** [comment]
Looks good, are you trying to get this in before branching today?

**Comment by CarolBusman** [comment]
> Looks good, are you trying to get this in before branching today?

Nop. I just want to get this in so that I can try to get a platformTools PR in.

**Pull request merged**